rosdep/python.yaml

```diff
@@ -5372,6 +5372,9 @@ python3-can:
   ubuntu:
     '*': [python3-can]
     xenial: null
+python3-catkin-lint:
+  debian: [python3-catkin-lint]
+  ubuntu: [python3-catkin-lint]
```

Comment:
The problem with both of these is that they are only available on newer distributions (since eoan on Ubuntu, since buston on Debian).  Since we still support older distributions, we'll need to do something more like:

```
  debian:
    '*': [python3-catkin-lint]
    stretch: null
  ubuntu:
    '*': [python3-catkin-lint]
    bionic: null
    xenial: null    
```